docker/run.sh

```
@@ -1,3 +1,11 @@
 #!/usr/bin/env bash
-cd /usr/src/wyoming-glados && 
-python3 __main__.py --uri 'tcp://0.0.0.0:10201' --debug
\ No newline at end of file
+
+# Navigate to the Wyoming-Glados directory
+cd /usr/src/wyoming-glados
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)